internal/hashing/reader_test.go

```diff
@@ -7,8 +7,29 @@ import (
 	"io"
 	"io/ioutil"
 	"testing"
+
+	rtest "github.com/restic/restic/internal/test"
 )
 
+type onlyReader struct {
+	rd io.Reader
```

Comment:
Here just a struct with `io.Reader` should be enough.

Reply:
is changed